src/test/resources/log4j.properties

```diff
@@ -0,0 +1,24 @@
+#
+# Biopet is built on top of GATK Queue for building bioinformatic
```

Comment:
Old license

Reply:
This should indeed be fixed, but this I think wrong in all tool for now